src/main/java/com/kylenicholls/stash/parameterizedbuilds/ciserver/CIServlet.java

```diff
@@ -61,13 +61,21 @@ protected void doPost(HttpServletRequest req, HttpServletResponse res)
 			CIServer ciServer = CIServerFactory.getServer(pathInfo, jenkins, req.getParameterMap(),
 					authContext.getCurrentUser(), projectService);
 
+			Map<String,String[]> parameters = req.getParameterMap();
```

Comment:
Is this just an artifact from debugging? If so, can you please remove this change?

Reply:
will definitively do that